apps/web/src/common/constants.ts

```diff
@@ -100,3 +100,36 @@ export const tooltipIcons: { [key in TooltipIconTypes]: TooltipIconProps } = {
     style: 'gray-icon',
   },
 };
+
+export const activityGapLegend = [
```

Comment:
I think we can re-use Leon's constants which had the same content

Reply:
Re used Leons  object.